From dd6329238ccf5737899acd6ee7893a27029a4552 Mon Sep 17 00:00:00 2001
From: Pankit Garg <pankit.garg@nxp.com>
Date: Fri, 11 Sep 2020 16:26:12 +0530
Subject: [PATCH 5/5] ls1046awrdb/RN_FFSSPPPN_1133_5577: Remove bin1 and bin2
 rcw config

Soc Part on ls1046awrdb do not support bin1 and bin2.
Modify RCW fields to support ls1046awrdb board.

Signed-off-by: Pankit Garg <pankit.garg@nxp.com>
---
 ...800_emmcboot.rcw => rcw_1200_emmcboot.rcw} |  14 +--
 .../rcw_1200_qspiboot.rcw                     |  15 ++-
 ...cw_1800_sdboot.rcw => rcw_1200_sdboot.rcw} |  15 +--
 .../rcw_1600_qspiboot.rcw                     | 115 -----------------
 .../rcw_1600_qspiboot_sben.rcw                | 118 ------------------
 .../rcw_1800_emmcboot_sben.rcw                | 116 -----------------
 .../rcw_1800_qspiboot.rcw                     | 114 -----------------
 .../rcw_1800_sdboot_sben.rcw                  | 116 -----------------
 8 files changed, 24 insertions(+), 599 deletions(-)
 rename ls1046awrdb/RN_FFSSPPPN_1133_5577/{rcw_1800_emmcboot.rcw => rcw_1200_emmcboot.rcw} (94%)
 rename ls1046awrdb/RN_FFSSPPPN_1133_5577/{rcw_1800_sdboot.rcw => rcw_1200_sdboot.rcw} (93%)
 delete mode 100644 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw
 delete mode 100644 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw
 delete mode 100644 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw
 delete mode 100644 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw
 delete mode 100644 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw

diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_emmcboot.rcw
similarity index 94%
rename from ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw
rename to ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_emmcboot.rcw
index c8235e2..82150f8 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_emmcboot.rcw
@@ -8,9 +8,9 @@
  * Sys Clock: 100 MHz
  * DDR_Refclock: 100 MHz
  *
- * Core		-- 1800 MHz (Mul 16)
- * Platform	-- 700 MHz (Mul 7)
- * DDR		-- 2100 MT/s (Mul 21)
+ * Core		-- 1200 MHz (Mul 12)
+ * Platform	-- 600 MHz (Mul 6)
+ * DDR		-- 1600 MT/s (Mul 16)
  * FMan		-- 700 MHz (CGA2 /2)
  * XFI		-- 156.25 MHz (10.3125G)
  * SGMII	-- 100 MHz (5G)
@@ -63,9 +63,9 @@
 
 #include <ls1046a.rcwi>
 
-SYS_PLL_RAT=7
-MEM_PLL_RAT=21
-CGA_PLL1_RAT=18
+SYS_PLL_RAT=6
+MEM_PLL_RAT=16
+CGA_PLL1_RAT=12
 CGA_PLL2_RAT=10
 SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
@@ -94,7 +94,7 @@ IFC_GRP_F_EXT=1
 IRQ_OUT=1
 TVDD_VSEL=0
 DVDD_VSEL=2
-EVDD_VSEL=0
+EVDD_VSEL=3
 IIC2_EXT=0
 SYSCLK_FREQ=600
 HWA_CGA_M2_CLK_SEL=1
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_qspiboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_qspiboot.rcw
index 37224ff..7a3d025 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_qspiboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_qspiboot.rcw
@@ -9,8 +9,8 @@
  * DDR_Refclock: 100 MHz
  *
  * Core		-- 1200 MHz (Mul 12)
- * Platform	-- 600 MHz (Mul 4)
- * DDR		-- 1300 MT/s (Mul 16)
+ * Platform	-- 600 MHz (Mul 6)
+ * DDR		-- 1600 MT/s (Mul 16)
  * FMan		-- 500 MHz (CGA2 /2)
  * XFI		-- 156.25 MHz (10.3125G)
  * SGMII	-- 100 MHz (5G)
@@ -64,11 +64,13 @@
 #include <ls1046a.rcwi>
 
 SYS_PLL_RAT=6
-MEM_PLL_RAT=13
+MEM_PLL_RAT=16
 CGA_PLL1_RAT=12
 CGA_PLL2_RAT=10
+SRDS_PLL_PD_S1=0
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
+SRDS_REFCLK_SEL_S2=1
 SRDS_PLL_REF_CLK_SEL_S1=1
 SRDS_PLL_REF_CLK_SEL_S2=0
 SRDS_DIV_PEX_S1=0
@@ -80,7 +82,7 @@ IFC_MODE=37
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -90,9 +92,9 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
-EVDD_VSEL=2
+EVDD_VSEL=3
 IIC2_EXT=1
 SYSCLK_FREQ=600
 HWA_CGA_M2_CLK_SEL=1
@@ -105,6 +107,7 @@ write 0x570600, 0x00000000
 write 0x570604, 0x40100000
 .end
 
+#include <qspi_speed.rcw>
 #include <cci_barrier_disable.rcw>
 #include <usb_phy_freq.rcw>
 #include <a009531.rcw>
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_sdboot.rcw
similarity index 93%
rename from ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw
rename to ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_sdboot.rcw
index 1a69ab3..067645c 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1200_sdboot.rcw
@@ -8,9 +8,9 @@
  * Sys Clock: 100 MHz
  * DDR_Refclock: 100 MHz
  *
- * Core		-- 1800 MHz (Mul 16)
- * Platform	-- 700 MHz (Mul 7)
- * DDR		-- 2100 MT/s (Mul 21)
+ * Core		-- 1200 MHz (Mul 12)
+ * Platform	-- 600 MHz (Mul 6)
+ * DDR		-- 1600 MT/s (Mul 16)
  * FMan		-- 700 MHz (CGA2 /2)
  * XFI		-- 156.25 MHz (10.3125G)
  * SGMII	-- 100 MHz (5G)
@@ -63,10 +63,11 @@
 
 #include <ls1046a.rcwi>
 
-SYS_PLL_RAT=7
-MEM_PLL_RAT=21
-CGA_PLL1_RAT=18
+SYS_PLL_RAT=6
+MEM_PLL_RAT=16
+CGA_PLL1_RAT=12
 CGA_PLL2_RAT=10
+SRDS_PLL_PD_S1=0
 SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
@@ -93,7 +94,7 @@ IFC_GRP_F_EXT=1
 IRQ_OUT=1
 TVDD_VSEL=0
 DVDD_VSEL=2
-EVDD_VSEL=2
+EVDD_VSEL=3
 IIC2_EXT=1
 SYSCLK_FREQ=600
 HWA_CGA_M2_CLK_SEL=1
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw
deleted file mode 100644
index e8a6c03..0000000
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw
+++ /dev/null
@@ -1,115 +0,0 @@
-/*
- * LS1046AWRDB RCW for SerDes Protocol 0x1133_5577
- *
- * 24G configuration -- two XFI + 2 SGMII + 3 PCIe
- *
- * Frequencies:
- *
- * Sys Clock: 100 MHz
- * DDR_Refclock: 100 MHz
- *
- * Core		-- 1600 MHz (Mul 16)
- * Platform	-- 600 MHz (Mul 6)
- * DDR		-- 1600 MT/s (Mul 21)
- * FMan		-- 700 MHz (CGA2 /2)
- * XFI		-- 156.25 MHz (10.3125G)
- * SGMII	-- 100 MHz (5G)
- * PCIE		-- 100 MHz (5G)
- * eSDHC	-- 1400 MHz (CGA2 /1)
- *
- * Hardware Accelerator Block Cluster Group A Mux Clock:
- *   FMan        - HWA_CGA_M1_CLK_SEL = 6 - Async mode, CGA PLL 2 /2 is clock
- *   eSDHC, QSPI - HWA_CGA_M2_CLK_SEL = 1 - Async mode, CGA PLL 2 /1 is clock
- *
- * Serdes Lanes vs Slot information
- *  Serdes1 Lane 0 (D) - XFI9, AQR107 PHY
- *  Serdes1 Lane 1 (C) - XFI10, SFP cage
- *  Serdes1 Lane 2 (B) - SGMII5, SGMII1 port
- *  Serdes1 Lane 3 (A) - SGMII6, SGMII2 port
- *
- *  Serdes2 Lane 0 (A) - PCIe1 Gen3 x1, Slot 1, mPCIe
- *  Serdes2 Lane 1 (B) - PCIe2 Gen3 x1, Slot 2
- *  Serdes2 Lane 2 (C D) - PCIe3 Gen3 x2, Slot 3
- *
- * PLL mapping: 2211_2221
- *
- * Serdes 1:
- *  PLL mapping: 2211
- *
- *  SRDS_PLL_REF_CLK_SEL_S1 : 0b'01
- *    SerDes 1, PLL1[160] : 0 - 100MHz for SGMII and PCIe
- *    SerDes 1, PLL2[161] : 1 - 156.25MHz for XFI
- *  SRDS_PLL_PD_S1 : 0b'0
- *    SerDes 1, PLL1 : 0 - not power down
- *    SerDes 1, PLL2 : 0 - not poewr down
- *  HWA_CGA_M1_CLK_SEL[224-226] : 6 - Cluster Group A PLL 2 /2 to FMan
- *
- * Serdes 2:
- *  PLL mapping: 2221
- *  SRDS_PLL_REF_CLK_SEL_S2 : 0b'00
- *    SerDes 2, PLL2[163] : 0 - 100MHz for PCIe
- *  SRDS_PLL_PD_S2 : 0b'00
- *    SerDes 2, PLL1 : 0 - not power down
- *    SerDes 2, PLL2 : 0 - not poewr down
- *  SRDS_DIV_PEX_S2 : 0b'01
- *    00 - train up to max rate of 8G
- *    01 - train up to max rate of 5G
- *    10 - train up to max rate of 2.5G
- *
- * DDR clock:
- * DDR_REFCLK_SEL : 1 - DDRCLK pin provides the reference clock to the DDR PLL
- *
- */
-
-#include <ls1046a.rcwi>
-
-SYS_PLL_RAT=6
-MEM_PLL_RAT=16
-CGA_PLL1_RAT=16
-CGA_PLL2_RAT=10
-SRDS_REFCLK_SEL_S2=1
-SRDS_PRTCL_S1=4403
-SRDS_PRTCL_S2=21879
-SRDS_PLL_REF_CLK_SEL_S1=1
-SRDS_PLL_REF_CLK_SEL_S2=0
-SRDS_DIV_PEX_S1=0
-SRDS_DIV_PEX_S2=0
-DDR_FDBK_MULT=2
-DDR_REFCLK_SEL=1
-PBI_SRC=4
-IFC_MODE=37
-EC2=1
-HWA_CGA_M1_CLK_SEL=6
-DRAM_LAT=1
-SPI_EXT=0
-UART_BASE=7
-IFC_GRP_A_BASE=1
-IFC_GRP_A_EXT=1
-IFC_GRP_D_BASE=1
-IFC_GRP_D_EXT=1
-IFC_GRP_E1_BASE=1
-IFC_GRP_E1_EXT=1
-IFC_GRP_F_EXT=1
-IRQ_OUT=1
-TVDD_VSEL=0
-DVDD_VSEL=2
-EVDD_VSEL=2
-IIC2_EXT=1
-SYSCLK_FREQ=600
-HWA_CGA_M2_CLK_SEL=1
-
-.pbi
-// set CLK_SEL for QSPI CONFIG Register
-write 0x57015c, 0x20100000
-// set SCFG_SCRATCHRW1 ansd SCFG_SCRATCHRW2 for boot location ptr
-write 0x570600, 0x00000000
-write 0x570604, 0x40100000
-.end
-
-// Errta A-008850 for ddr controller for barrier transaction
-#include <cci_barrier_disable.rcw>
-// Set USB PHY PLL for 100MHz
-#include <usb_phy_freq.rcw>
-// Errata A-010477 and  A-008851 for PCI Express Gen3 link training
-#include <../ls1046ardb/pex_gen3_link.rcw>
-#include <qspi_endianness.rcw>
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw
deleted file mode 100644
index e0c5b9d..0000000
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw
+++ /dev/null
@@ -1,118 +0,0 @@
-/*
- * LS1046AWRDB RCW for SerDes Protocol 0x1133_5577
- *
- * 24G configuration -- two XFI + 2 SGMII + 3 PCIe
- *
- * Frequencies:
- *
- * Sys Clock: 100 MHz
- * DDR_Refclock: 100 MHz
- *
- * Core		-- 1600 MHz (Mul 16)
- * Platform	-- 600 MHz (Mul 6)
- * DDR		-- 1600 MT/s (Mul 21)
- * FMan		-- 700 MHz (CGA2 /2)
- * XFI		-- 156.25 MHz (10.3125G)
- * SGMII	-- 100 MHz (5G)
- * PCIE		-- 100 MHz (5G)
- * eSDHC	-- 1400 MHz (CGA2 /1)
- *
- * Hardware Accelerator Block Cluster Group A Mux Clock:
- *   FMan        - HWA_CGA_M1_CLK_SEL = 6 - Async mode, CGA PLL 2 /2 is clock
- *   eSDHC, QSPI - HWA_CGA_M2_CLK_SEL = 1 - Async mode, CGA PLL 2 /1 is clock
- *
- * Serdes Lanes vs Slot information
- *  Serdes1 Lane 0 (D) - XFI9, AQR107 PHY
- *  Serdes1 Lane 1 (C) - XFI10, SFP cage
- *  Serdes1 Lane 2 (B) - SGMII5, SGMII1 port
- *  Serdes1 Lane 3 (A) - SGMII6, SGMII2 port
- *
- *  Serdes2 Lane 0 (A) - PCIe1 Gen3 x1, Slot 1, mPCIe
- *  Serdes2 Lane 1 (B) - PCIe2 Gen3 x1, Slot 2
- *  Serdes2 Lane 2 (C D) - PCIe3 Gen3 x2, Slot 3
- *
- * PLL mapping: 2211_2221
- *
- * Serdes 1:
- *  PLL mapping: 2211
- *
- *  SRDS_PLL_REF_CLK_SEL_S1 : 0b'01
- *    SerDes 1, PLL1[160] : 0 - 100MHz for SGMII and PCIe
- *    SerDes 1, PLL2[161] : 1 - 156.25MHz for XFI
- *  SRDS_PLL_PD_S1 : 0b'0
- *    SerDes 1, PLL1 : 0 - not power down
- *    SerDes 1, PLL2 : 0 - not poewr down
- *  HWA_CGA_M1_CLK_SEL[224-226] : 6 - Cluster Group A PLL 2 /2 to FMan
- *
- * Serdes 2:
- *  PLL mapping: 2221
- *  SRDS_PLL_REF_CLK_SEL_S2 : 0b'00
- *    SerDes 2, PLL2[163] : 0 - 100MHz for PCIe
- *  SRDS_PLL_PD_S2 : 0b'00
- *    SerDes 2, PLL1 : 0 - not power down
- *    SerDes 2, PLL2 : 0 - not poewr down
- *  SRDS_DIV_PEX_S2 : 0b'01
- *    00 - train up to max rate of 8G
- *    01 - train up to max rate of 5G
- *    10 - train up to max rate of 2.5G
- *
- * DDR clock:
- * DDR_REFCLK_SEL : 1 - DDRCLK pin provides the reference clock to the DDR PLL
- *
- */
-
-#include <ls1046a.rcwi>
-
-SYS_PLL_RAT=6
-MEM_PLL_RAT=16
-CGA_PLL1_RAT=16
-CGA_PLL2_RAT=10
-SRDS_REFCLK_SEL_S2=1
-SRDS_PRTCL_S1=4403
-SRDS_PRTCL_S2=21879
-SRDS_PLL_REF_CLK_SEL_S1=1
-SRDS_PLL_REF_CLK_SEL_S2=0
-SRDS_DIV_PEX_S1=0
-SRDS_DIV_PEX_S2=0
-DDR_FDBK_MULT=2
-DDR_REFCLK_SEL=1
-PBI_SRC=4
-SB_EN=1
-BOOT_HO=1
-IFC_MODE=37
-EC2=1
-HWA_CGA_M1_CLK_SEL=6
-DRAM_LAT=1
-SPI_EXT=0
-UART_BASE=7
-IFC_GRP_A_BASE=1
-IFC_GRP_A_EXT=1
-IFC_GRP_D_BASE=1
-IFC_GRP_D_EXT=1
-IFC_GRP_E1_BASE=1
-IFC_GRP_E1_EXT=1
-IFC_GRP_F_EXT=1
-IRQ_OUT=1
-TVDD_VSEL=0
-DVDD_VSEL=2
-EVDD_VSEL=2
-IIC2_EXT=1
-SYSCLK_FREQ=600
-HWA_CGA_M2_CLK_SEL=1
-
-.pbi
-// U-boot header address
-write 0xee0200, 0x406c0000
-// set CLK_SEL for QSPI CONFIG Register
-write 0x57015c, 0x20100000
-// set SCFG_SCRATCHRW1 ansd SCFG_SCRATCHRW2 for boot location ptr
-write 0x570600, 0x00000000
-write 0x570604, 0x40100000
-.end
-
-#include <qspi_speed.rcw>
-#include <cci_barrier_disable.rcw>
-#include <usb_phy_freq.rcw>
-#include <a009531.rcw>
-#include <../ls1046ardb/pex_gen3_link.rcw>
-#include <qspi_endianness.rcw>
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw
deleted file mode 100644
index e8000bd..0000000
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw
+++ /dev/null
@@ -1,116 +0,0 @@
-/*
- * LS1046AWRDB RCW for SerDes Protocol 0x1133_5577
- *
- * 24G configuration -- two XFI + 2 SGMII + 3 PCIe
- *
- * Frequencies:
- *
- * Sys Clock: 100 MHz
- * DDR_Refclock: 100 MHz
- *
- * Core		-- 1800 MHz (Mul 16)
- * Platform	-- 700 MHz (Mul 7)
- * DDR		-- 2100 MT/s (Mul 21)
- * FMan		-- 700 MHz (CGA2 /2)
- * XFI		-- 156.25 MHz (10.3125G)
- * SGMII	-- 100 MHz (5G)
- * PCIE		-- 100 MHz (5G)
- * eSDHC	-- 1400 MHz (CGA2 /1)
- *
- * Hardware Accelerator Block Cluster Group A Mux Clock:
- *   FMan        - HWA_CGA_M1_CLK_SEL = 6 - Async mode, CGA PLL 2 /2 is clock
- *   eSDHC, QSPI - HWA_CGA_M2_CLK_SEL = 1 - Async mode, CGA PLL 2 /1 is clock
- *
- * Serdes Lanes vs Slot information
- *  Serdes1 Lane 0 (D) - XFI9, AQR107 PHY
- *  Serdes1 Lane 1 (C) - XFI10, SFP cage
- *  Serdes1 Lane 2 (B) - SGMII5, SGMII1 port
- *  Serdes1 Lane 3 (A) - SGMII6, SGMII2 port
- *
- *  Serdes2 Lane 0 (A) - PCIe1 Gen3 x1, Slot 1, mPCIe
- *  Serdes2 Lane 1 (B) - PCIe2 Gen3 x1, Slot 2
- *  Serdes2 Lane 2 (C D) - PCIe3 Gen3 x2, Slot 3
- *
- * PLL mapping: 2211_2221
- *
- * Serdes 1:
- *  PLL mapping: 2211
- *
- *  SRDS_PLL_REF_CLK_SEL_S1 : 0b'01
- *    SerDes 1, PLL1[160] : 0 - 100MHz for SGMII and PCIe
- *    SerDes 1, PLL2[161] : 1 - 156.25MHz for XFI
- *  SRDS_PLL_PD_S1 : 0b'0
- *    SerDes 1, PLL1 : 0 - not power down
- *    SerDes 1, PLL2 : 0 - not poewr down
- *  HWA_CGA_M1_CLK_SEL[224-226] : 6 - Cluster Group A PLL 2 /2 to FMan
- *
- * Serdes 2:
- *  PLL mapping: 2221
- *  SRDS_PLL_REF_CLK_SEL_S2 : 0b'00
- *    SerDes 2, PLL2[163] : 0 - 100MHz for PCIe
- *  SRDS_PLL_PD_S2 : 0b'00
- *    SerDes 2, PLL1 : 0 - not power down
- *    SerDes 2, PLL2 : 0 - not poewr down
- *  SRDS_DIV_PEX_S2 : 0b'01
- *    00 - train up to max rate of 8G
- *    01 - train up to max rate of 5G
- *    10 - train up to max rate of 2.5G
- *
- * DDR clock:
- * DDR_REFCLK_SEL : 1 - DDRCLK pin provides the reference clock to the DDR PLL
- *
- */
-
-#include <ls1046a.rcwi>
-
-SYS_PLL_RAT=7
-MEM_PLL_RAT=21
-CGA_PLL1_RAT=18
-CGA_PLL2_RAT=10
-SRDS_REFCLK_SEL_S2=1
-SRDS_PRTCL_S1=4403
-SRDS_PRTCL_S2=21879
-SRDS_PLL_REF_CLK_SEL_S1=1
-SRDS_PLL_REF_CLK_SEL_S2=0
-SRDS_DIV_PEX_S1=0
-SRDS_DIV_PEX_S2=0
-DDR_FDBK_MULT=2
-DDR_REFCLK_SEL=1
-PBI_SRC=6
-SB_EN=1
-BOOT_HO=1
-IFC_MODE=64
-HWA_CGA_M1_CLK_SEL=6
-DRAM_LAT=1
-SPI_EXT=0
-SPI_BASE=1
-UART_BASE=7
-IFC_GRP_A_BASE=1
-IFC_GRP_A_EXT=1
-IFC_GRP_D_BASE=1
-IFC_GRP_D_EXT=1
-IFC_GRP_E1_BASE=1
-IFC_GRP_E1_EXT=1
-IFC_GRP_F_EXT=1
-IRQ_OUT=1
-TVDD_VSEL=0
-DVDD_VSEL=2
-EVDD_VSEL=0
-IIC2_EXT=0
-SYSCLK_FREQ=600
-HWA_CGA_M2_CLK_SEL=1
-
-.pbi
-// set SCFG_SCRATCHRW1 ansd SCFG_SCRATCHRW2 for boot location ptr
-write 0x570600, 0x00000000
-write 0x570604, 0x10000000
-.end
-
-#include <cci_barrier_disable.rcw>
-#include <usb_phy_freq.rcw>
-#include <pex_gen3_link.rcw>
-
-.pbi
-// flush PBI data
-write 0x6100c0, 0x000fffff
-.end
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw
deleted file mode 100644
index 2c9fd44..0000000
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw
+++ /dev/null
@@ -1,114 +0,0 @@
-/*
- * LS1046AWRDB RCW for SerDes Protocol 0x1133_5577
- *
- * 24G configuration -- two XFI + 2 SGMII + 3 PCIe
- *
- * Frequencies:
- *
- * Sys Clock: 100 MHz
- * DDR_Refclock: 100 MHz
- *
- * Core		-- 1800 MHz (Mul 16)
- * Platform	-- 700 MHz (Mul 7)
- * DDR		-- 2100 MT/s (Mul 21)
- * FMan		-- 800 MHz (CGA2 /2)
- * XFI		-- 156.25 MHz (10.3125G)
- * SGMII	-- 100 MHz (5G)
- * PCIE		-- 100 MHz (5G)
- * eSDHC	-- 1400 MHz (CGA2 /1)
- *
- * Hardware Accelerator Block Cluster Group A Mux Clock:
- *   FMan        - HWA_CGA_M1_CLK_SEL = 6 - Async mode, CGA PLL 2 /2 is clock
- *   eSDHC, QSPI - HWA_CGA_M2_CLK_SEL = 1 - Async mode, CGA PLL 2 /1 is clock
- *
- * Serdes Lanes vs Slot information
- *  Serdes1 Lane 0 (D) - XFI9, AQR107 PHY
- *  Serdes1 Lane 1 (C) - XFI10, SFP cage
- *  Serdes1 Lane 2 (B) - SGMII5, SGMII1 port
- *  Serdes1 Lane 3 (A) - SGMII6, SGMII2 port
- *
- *  Serdes2 Lane 0 (A) - PCIe1 Gen3 x1, Slot 1, mPCIe
- *  Serdes2 Lane 1 (B) - PCIe2 Gen3 x1, Slot 2
- *  Serdes2 Lane 2 (C D) - PCIe3 Gen3 x2, Slot 3
- *
- * PLL mapping: 2211_2221
- *
- * Serdes 1:
- *  PLL mapping: 2211
- *
- *  SRDS_PLL_REF_CLK_SEL_S1 : 0b'01
- *    SerDes 1, PLL1[160] : 0 - 100MHz for SGMII and PCIe
- *    SerDes 1, PLL2[161] : 1 - 156.25MHz for XFI
- *  SRDS_PLL_PD_S1 : 0b'0
- *    SerDes 1, PLL1 : 0 - not power down
- *    SerDes 1, PLL2 : 0 - not poewr down
- *  HWA_CGA_M1_CLK_SEL[224-226] : 6 - Cluster Group A PLL 2 /2 to FMan
- *
- * Serdes 2:
- *  PLL mapping: 2221
- *  SRDS_PLL_REF_CLK_SEL_S2 : 0b'00
- *    SerDes 2, PLL2[163] : 0 - 100MHz for PCIe
- *  SRDS_PLL_PD_S2 : 0b'00
- *    SerDes 2, PLL1 : 0 - not power down
- *    SerDes 2, PLL2 : 0 - not poewr down
- *  SRDS_DIV_PEX_S2 : 0b'01
- *    00 - train up to max rate of 8G
- *    01 - train up to max rate of 5G
- *    10 - train up to max rate of 2.5G
- *
- * DDR clock:
- * DDR_REFCLK_SEL : 1 - DDRCLK pin provides the reference clock to the DDR PLL
- *
- */
-
-#include <ls1046a.rcwi>
-
-SYS_PLL_RAT=7
-MEM_PLL_RAT=21
-CGA_PLL1_RAT=18
-CGA_PLL2_RAT=10
-SRDS_REFCLK_SEL_S2=1
-SRDS_PRTCL_S1=4403
-SRDS_PRTCL_S2=21879
-SRDS_PLL_REF_CLK_SEL_S1=1
-SRDS_PLL_REF_CLK_SEL_S2=0
-SRDS_DIV_PEX_S1=0
-SRDS_DIV_PEX_S2=0
-DDR_FDBK_MULT=2
-DDR_REFCLK_SEL=1
-PBI_SRC=4
-IFC_MODE=37
-EC2=1
-HWA_CGA_M1_CLK_SEL=6
-DRAM_LAT=1
-SPI_EXT=0
-UART_BASE=7
-IFC_GRP_A_BASE=1
-IFC_GRP_A_EXT=1
-IFC_GRP_D_BASE=1
-IFC_GRP_D_EXT=1
-IFC_GRP_E1_BASE=1
-IFC_GRP_E1_EXT=1
-IFC_GRP_F_EXT=1
-IRQ_OUT=1
-TVDD_VSEL=0
-DVDD_VSEL=2
-EVDD_VSEL=2
-IIC2_EXT=1
-SYSCLK_FREQ=600
-HWA_CGA_M2_CLK_SEL=1
-
-.pbi
-// set CLK_SEL for QSPI CONFIG Register
-write 0x57015c, 0x30100000
-// set SCFG_SCRATCHRW1 ansd SCFG_SCRATCHRW2 for boot location ptr
-write 0x570600, 0x00000000
-write 0x570604, 0x40100000
-.end
-
-#include <qspi_speed.rcw>
-#include <cci_barrier_disable.rcw>
-#include <usb_phy_freq.rcw>
-// Errata A-010477 and  A-008851 for PCI Express Gen3 link training
-#include <../ls1046ardb/pex_gen3_link.rcw>
-#include <qspi_endianness.rcw>
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw
deleted file mode 100644
index 5d435ec..0000000
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw
+++ /dev/null
@@ -1,116 +0,0 @@
-/*
- * LS1046AWRDB RCW for SerDes Protocol 0x1133_5577
- *
- * 24G configuration -- two XFI + 2 SGMII + 3 PCIe
- *
- * Frequencies:
- *
- * Sys Clock: 100 MHz
- * DDR_Refclock: 100 MHz
- *
- * Core		-- 1800 MHz (Mul 16)
- * Platform	-- 700 MHz (Mul 7)
- * DDR		-- 2100 MT/s (Mul 21)
- * FMan		-- 700 MHz (CGA2 /2)
- * XFI		-- 156.25 MHz (10.3125G)
- * SGMII	-- 100 MHz (5G)
- * PCIE		-- 100 MHz (5G)
- * eSDHC	-- 1400 MHz (CGA2 /1)
- *
- * Hardware Accelerator Block Cluster Group A Mux Clock:
- *   FMan        - HWA_CGA_M1_CLK_SEL = 6 - Async mode, CGA PLL 2 /2 is clock
- *   eSDHC, QSPI - HWA_CGA_M2_CLK_SEL = 1 - Async mode, CGA PLL 2 /1 is clock
- *
- * Serdes Lanes vs Slot information
- *  Serdes1 Lane 0 (D) - XFI9, AQR107 PHY
- *  Serdes1 Lane 1 (C) - XFI10, SFP cage
- *  Serdes1 Lane 2 (B) - SGMII5, SGMII1 port
- *  Serdes1 Lane 3 (A) - SGMII6, SGMII2 port
- *
- *  Serdes2 Lane 0 (A) - PCIe1 Gen3 x1, Slot 1, mPCIe
- *  Serdes2 Lane 1 (B) - PCIe2 Gen3 x1, Slot 2
- *  Serdes2 Lane 2 (C D) - PCIe3 Gen3 x2, Slot 3
- *
- * PLL mapping: 2211_2221
- *
- * Serdes 1:
- *  PLL mapping: 2211
- *
- *  SRDS_PLL_REF_CLK_SEL_S1 : 0b'01
- *    SerDes 1, PLL1[160] : 0 - 100MHz for SGMII and PCIe
- *    SerDes 1, PLL2[161] : 1 - 156.25MHz for XFI
- *  SRDS_PLL_PD_S1 : 0b'0
- *    SerDes 1, PLL1 : 0 - not power down
- *    SerDes 1, PLL2 : 0 - not poewr down
- *  HWA_CGA_M1_CLK_SEL[224-226] : 6 - Cluster Group A PLL 2 /2 to FMan
- *
- * Serdes 2:
- *  PLL mapping: 2221
- *  SRDS_PLL_REF_CLK_SEL_S2 : 0b'00
- *    SerDes 2, PLL2[163] : 0 - 100MHz for PCIe
- *  SRDS_PLL_PD_S2 : 0b'00
- *    SerDes 2, PLL1 : 0 - not power down
- *    SerDes 2, PLL2 : 0 - not poewr down
- *  SRDS_DIV_PEX_S2 : 0b'01
- *    00 - train up to max rate of 8G
- *    01 - train up to max rate of 5G
- *    10 - train up to max rate of 2.5G
- *
- * DDR clock:
- * DDR_REFCLK_SEL : 1 - DDRCLK pin provides the reference clock to the DDR PLL
- *
- */
-
-#include <ls1046a.rcwi>
-
-SYS_PLL_RAT=7
-MEM_PLL_RAT=21
-CGA_PLL1_RAT=18
-CGA_PLL2_RAT=10
-SRDS_REFCLK_SEL_S2=1
-SRDS_PRTCL_S1=4403
-SRDS_PRTCL_S2=21879
-SRDS_PLL_REF_CLK_SEL_S1=1
-SRDS_PLL_REF_CLK_SEL_S2=0
-SRDS_DIV_PEX_S1=0
-SRDS_DIV_PEX_S2=0
-DDR_FDBK_MULT=2
-DDR_REFCLK_SEL=1
-PBI_SRC=6
-SB_EN=1
-BOOT_HO=1
-IFC_MODE=64
-EC2=1
-HWA_CGA_M1_CLK_SEL=6
-DRAM_LAT=1
-SPI_EXT=0
-UART_BASE=7
-IFC_GRP_A_BASE=1
-IFC_GRP_A_EXT=1
-IFC_GRP_D_BASE=1
-IFC_GRP_D_EXT=1
-IFC_GRP_E1_BASE=1
-IFC_GRP_E1_EXT=1
-IFC_GRP_F_EXT=1
-IRQ_OUT=1
-TVDD_VSEL=0
-DVDD_VSEL=2
-EVDD_VSEL=2
-IIC2_EXT=1
-SYSCLK_FREQ=600
-HWA_CGA_M2_CLK_SEL=1
-
-.pbi
-// set SCFG_SCRATCHRW1 ansd SCFG_SCRATCHRW2 for boot location ptr
-write 0x570600, 0x00000000
-write 0x570604, 0x10000000
-.end
-
-#include <cci_barrier_disable.rcw>
-#include <usb_phy_freq.rcw>
-#include <pex_gen3_link.rcw>
-
-.pbi
-// flush PBI data
-write 0x6100c0, 0x000fffff
-.end
-- 
2.17.1

